src/lua/CommunityBalanceMod/FileHooks.lua

```diff
@@ -3,6 +3,11 @@ local defaultConfig = {
     build_tag = "dev"
 }
 
+local kVersion = "1.5.2"
```

Comment:
Can we get this from g_communityBalanceModConfig instead of hardcoding it?

Reply:
Done.